usr/src/uts/common/sys/fs/zfs.h

```diff
@@ -153,6 +153,10 @@ typedef enum {
 	ZFS_PROP_FILESYSTEM_COUNT,
 	ZFS_PROP_SNAPSHOT_COUNT,
 	ZFS_PROP_REDUNDANT_METADATA,
+	ZFS_PROP_ENCRYPTION,
+	ZFS_PROP_SALT,
+	ZFS_PROP_KEYSOURCE,
+	ZFS_PROP_KEYSTATUS,
```

Comment:
these were added in the middle because master added more properties while I was still working on the patch. I will fix this now.